src/components/Registers.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Category from './Category';
+
+const RegisterPanel = ({ registers }) => {
+  const rs = registers.map((reg, i) => (
+    <li key={`{register-${i.toString()}}`}>
+      x
```

Comment:
is there a way we could clean this up slightly so its not over 4 lines?

Reply:
lint wanted me to put each element in the <li> tag on its own line, which i also think looks ugly. but it wont let me put it on one line, or any number where there are more than two elements like "x" and "{ i }" on the same line